ui/app/src/CHANGELOG.md

```diff
@@ -23,6 +26,14 @@
   * `getUserPermissions`
     * Use `fetchMyPermissions` instead
   * [services/contentTypes] Add `fetchContentType` service
+  * `services/content/insertComponent`: function now requires the parent document content type and the path argument moves to being earlier in the argument list.
+    The shifting of the arguments seeks a more coherent argument order, grouping parent-related arguments first, followed by inserted instance related arguments, and finally supportive arguments last.
+    * **Previously**: siteId, parentModelId, parentFieldId, targetIndex, **_insertedItemContentType_**, **_insertedContentInstance_**, _**parentDocPath**_, isSharedInstance, shouldSerializeValueFn?
+    * **Now**: siteId, _**parentDocPath**_, parentModelId, parentFieldId, targetIndex, _**parentContentType**_, _**insertedContentInstance**_, _**insertedItemContentType**_, isSharedInstance, shouldSerializeValueFn?
+  * `services/content/insertInstance`: function now requires the parent document content type and the path argument moves to being earlier in the argument list.
+    The shifting of the arguments seeks a more coherent argument order, grouping parent-related arguments first, followed by inserted instance related arguments, and finally supportive arguments last.
+    * **Previously**: siteId, parentModelId, parentFieldId, targetIndex, insertedInstance, _**parentDocPath**_, datasource?
+    * **Now**: siteId, _**parentDocPath**_, parentModelId, parentFieldId, targetIndex, _**parentContentType**_, insertedInstance, datasource?
```

Comment:
These were on the 4.1.x changelog. I reckon we just bring the changelog from 4.1.x here only include changes from **this** release.

Reply:
Reverted the CHANGELOG update. 4.1.3 CHANGELOG updates (4.1.x addition) added in [here](https://github.com/craftercms/studio-ui/pull/3748)